src/java/org/apache/cassandra/db/rows/NativeCell.java

```diff
@@ -194,20 +195,20 @@ public long unsharedHeapSizeExcludingData()
 
     public long offHeapSize()
     {
-        long size = offHeapSizeWithoutPath(MemoryUtil.getInt(peer + LENGTH));
+        long size = offHeapSizeWithoutPath(NativeEndianMemoryUtil.getInt(peer + LENGTH));
         if (hasPath())
-            size += 4 + MemoryUtil.getInt(peer + size);
+            size += 4 + NativeEndianMemoryUtil.getInt(peer + size);
         return size;
     }
 
     private boolean hasPath()
     {
-        return MemoryUtil.getByte(peer+ HAS_CELLPATH) != 0;
+        return MemoryUtil.getByte(peer + HAS_CELLPATH) != 0;
```

Comment:
Nit: this should also use `NativeEndianMemoryUtil` for consistency.

Reply:
I was not sure here, for one side - yes, it is better for consistency point of view, for another side I wanted to highlight that byte operations are endian agnostic..  I will switch to NativeEndianMemoryUtil usage